### PR TITLE
Clarification about the If-Else block.

### PR DESCRIPTION
**Pull request by BlueJayBird**
Just some added clarification about how when the condition for the 'If' section is not met it performs the 'else' block by default

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Comment by RandellDawson** [comment]
Closed and Reopened this PR to attempt to resolve a specific Travis build failure.

File: guide/english/csharp/if-else-statement/index.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ title: If Else Statement
 # If Else Statement
 
 The If-Else statement executes a block of code depending on whether your precondition is fullfilled or not.
```
Comment (thecodingaviator) [inline guide/english/csharp/if-else-statement/index.md]:
```suggestion
The `if else` statement executes a block of code depending on whether your precondition is fulfilled or not.
```

File: guide/english/csharp/if-else-statement/index.md [hunk 2]
```diff
@@ -5,6 +5,7 @@ title: If Else Statement
...
+If the 'If' condition is not met then it performs the 'Else' block by default.
```
Comment (thecodingaviator) [inline guide/english/csharp/if-else-statement/index.md]:
```suggestion
If the `if` condition is not met then it performs the `else` block. 
```

**Review by thecodingaviator — CHANGES_REQUESTED** [review]

**Comment by RandellDawson** [comment]
@thecodingaviator This one has gone stale.

**Comment by thecodingaviator** [comment]
Hi @BlueJayBird 

As there has been no response from your end for a while, we are closing this PR.

Feel free to request this PR to be reopened, if you still wish to make the changes you originally added.  Also, remember to update your fork, so you have the latest version of master in case it has changed significantly.

Happy Coding! ✈️

**Pull request closed without merging**